protostar/starknet/cheaters/contracts.py

```diff
@@ -280,7 +282,7 @@ async def call(
             function_name=function_name,
             calldata=calldata,
         )
-        entry_point = ExecuteEntryPoint.create_for_testing(
+        entry_point = CheatableExecuteEntryPoint.create_for_testing(
```

Comment:
I am aware of this issue and working on a solution in parallel with you - i think a new entrypoint class would be appropriate since that one has a lot of stuff that we don't need

Reply:
I fixed it here: https://github.com/software-mansion/protostar/pull/1367